tests/cases/test_timestamp.py

```diff
@@ -0,0 +1,39 @@
+from sqlmodel import Session
+from app.models.cases import Case
+from app.models.case_types import CaseTypes
+from datetime import datetime, UTC
+
+
+def test_timezone():
+    case = Case(case_type=CaseTypes.CLA)
+    assert case.created_at.tzinfo == UTC
+
+
+def test_created_at():
+    before_case_creation = datetime.now(UTC)
+    case = Case(case_type=CaseTypes.CLA)
+    after_case_creation = datetime.now(UTC)
+    assert before_case_creation < case.created_at < after_case_creation
```

Comment:
Should we use <= here as well?

Reply:
no we shouldnt, as the time shouldnt be the same, so `before_case_creation` should be less than (and not equal too as they are not created at the same time `case.created_at` and then `case.created_at` should be less than `after_case_creation` as once again they are not created at the same and we are confirming that its both less and not equal to.

Reply:
How is this different from the test after where we use <= - just considering edge case if it's all within the same millisecond if that's even possible.

Reply:
discussed and resolve